float64 value_1         # First operand
float64 value_2         # Second operand
string operation        # Mathematical operation (e.g., "add", "subtract", "multiply", "divide")
---
float64 result          # Computed result of the operation
---
float64 time_remaining  # Estimated time left for processing
